vinyl/tests/Functions.php

```diff
@@ -7,8 +7,8 @@ class Functions extends TestCase {
     public function testSuccessdisplayRecords()
     {
         //expected result of the test
-        $expected = '<div><h3>Cuthbert</h3><h4>Pizza</h4><p>7</p></div>';
-        
+        $expected = '<div><section><h3>Cuthbert</h3><h4>Pizza</h4><p>7</p></section></div>';
```

Comment:
Almost there! The section will also include a class, so I think the opening section tag here should be:

`<section class ="recordsContent">` instead of just `<section>`